scripts/setupPythonEnvironment.bash

```diff
@@ -3,8 +3,9 @@
 
 # Configuration
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-PACKAGE_DIR=$SCRIPT_DIR/../src/coreComponents/python/modules
+PACKAGE_DIR=$(mktemp -d)
```

Comment:
Is this really is a `PACKAGE_DIR` or more `TMP_CLONE_DIR`?
Also, does it make sense to optionally provide a `--tmpdir` as CLI argument that could be the `CMAKE_BUILD_DIR`? I think that it could be nice no to leave the `build dir`.

Reply:
Sure, I was just avoiding the re-name.  I agree that a CLI arg would be a good idea here